Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/direct/messagemonitoring/impl/MessageMonitoringAPI.java

```diff
@@ -310,80 +306,65 @@ public void clearCache() {
      * @return List
      */
     public List<MonitoredMessage> getAllFailedMessages() {
-        //loop through the list and find all the pending messages
-        List<MonitoredMessage> failedMessages = new ArrayList<>();
-        //loop through the cache
-        for (MonitoredMessage trackMessage : messageMonitoringCache.values()) {
+        // loop through the list and find all the pending messages
+        final List<MonitoredMessage> failedMessages = new ArrayList<>();
+        // loop through the cache
+        for (final MonitoredMessage trackMessage : messageMonitoringCache.values()) {
             if (trackMessage.getStatus().equals(STATUS_ERROR)) {
                 failedMessages.add(trackMessage);
             } else if (trackMessage.getStatus().equals(STATUS_PENDING)) {
-                //if its pending & if its elapsed then
-                //change the status to Error
+                // if its pending & if its elapsed then
```

Comment:
Why we need an else if (trackMessage.getStatus().equals(STATUS_PENDING)) block, when no processing is done in this block.